src/CommunityToolkit.Maui.Core/Capabilities/StatusBar/StatusBar.net.cs

```diff
@@ -0,0 +1,8 @@
+namespace CommunityToolkit.Maui.Core.Capabilities;
+
+static partial class StatusBar
+{
+	static void PlatformSetColor(Color color) => throw new NotImplementedException();
```

Comment:
Maybe we should log a message that it is not supported, like you have in Android (IsSupported method). It may be a blocker for devs who want to release on Windows.

or use SuppressException options in Core.Options

Reply:
I added the attribute telling this is not supported on windows. I'm pretty sure I did it for the Behavior, not sure for the Class in core, need to double check that

Reply:
Yes, it will require devs to add #if !Windows which may be annoying.
Debug.WriteLine() is more friendly. Anyway we add in docs that it is not supported on windows

Reply:
`Debug.WriteLine` will print nothing in the package :/

Reply:
Debug.WriteLine is terrible :)